core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -238,9 +247,10 @@ object H2OContext extends Logging{
     * @param conf H2O configuration
     * @return H2O Context
     */
-  def getOrCreate(sc: SparkContext, conf: H2OConf): H2OContext = synchronized {
+  def getOrCreate(sc: SparkContext, conf: H2OConf)(implicit sqlContext: SQLContext): H2OContext = synchronized {
```

Comment:
This will break backward compatibility, can we leave the old signature and simply call this one from it like you're doing with `getOrCreate(SparkContext)`?